examples/vertx-opentracing/pom.xml

```diff
@@ -22,7 +22,7 @@
 
   <groupId>org.hawkular.apm</groupId>
   <artifactId>hawkular-apm-examples-vertx-opentracing</artifactId>
-  <version>0.1-SNAPSHOT</version>
+  <version>0.11.1.Final-SNAPSHOT</version>
```

Comment:
What was the real issue (I think probably parent wasn't available at local repo right?)? In this case to build the image successfully you have to include all APM build it from the root (it will build the example too). I would prefer this solution.